gtsfm/runner/run_scene_optimizer.py

```diff
@@ -55,6 +55,24 @@ def run_scene_optimizer(args) -> None:
         default=20,
         help="maximum number of consecutive frames to consider for matching/co-visibility",
     )
+    parser.add_argument(
+        "--num_workers",
+        type=int,
+        default=1,
+        help="Number of workers to start (processes, by default)",
+    )
+    parser.add_argument(
+        "--threads_per_worker",
+        type=int,
+        default=1,
+        help="Number of threads per each worker",
+    )
+    parser.add_argument(
+        "--config_name",
+        type=str,
+        default="deep_front_end.yaml",
+        help="Choose default_lund_door_set1_config.yaml or deep_front_end.yaml",
```

Comment:
maybe time to rename `default_lund_door_set1_config`?

Reply:
I went ahead and changed the name to `sift_front_end.yaml` @ayushbaid -- let me know how that looks